src/Ensconce.Console/Arguments.cs

```diff
@@ -112,6 +114,11 @@ private static OptionSet GetOptionSet()
                     ,"The entry to be made in the repository field in the RoundhousE version table. If not provided defaults to an empty string. NOTE! Ignored if databaseName is not provided.",
                     s => DatabaseRepository = string.IsNullOrEmpty(s) ? DatabaseRepository : s
                 },
+                {
+                    "databaseCommandTimeout=",
+                    "Database Command Timeout period in seconds. If not provided defaults to 30s.",
+                    s => DatabaseCommandTimeout = string.IsNullOrWhiteSpace(s) ? TimeSpan.FromSeconds(30) : TimeSpan.FromSeconds(Convert.ToInt32(s))
```

Comment:
Hi @IgnasLaukineitis, have discussed with Chris, and we'd also like this value to be settable using an Environment Variable (see line 64 in this file for an example).

You'd still be able to override using the command line as you've done, but it means we can change it globally quite easily

Thanks